partcad-ide-vscode/bundled/tool/lsp_server.py

```
@@ -0,0 +1,1590 @@
+    try:
+        do_load_package_contents(args)
+    except partcad.exception.NeedsUpdateException as e:
+        LSP_SERVER.send_notification("?/partcad/needsUpdate")
+    except Exception as e:
+        LSP_SERVER.send_notification(
+            "?/partcad/error",
+            "Failed to load package contents: %s" % e.with_traceback(None),
+        )
+
+
+def do_load_package_contents(args=list()) -> None:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)